.github/workflows/build.yml

```
@@ -277,36 +278,37 @@ jobs:
           context: .
           file: ./Dockerfile
           outputs: type=docker,dest=dist/image.tar
-          tags: ${{ env.IMAGE_NAME }}:latest  # not to be pushed
+          # not to be pushed
+          tags: ${{ env.IMAGE_NAME }}:latest
```

We prefer to avoid making unnecessary changes from the [skeleton](https://github.com/cisagov/skeleton-docker/blob/9cea64da4a1379cdde3c509a09fc548f0da8b034/.github/workflows/build.yml#L264):
```suggestion
          tags: ${{ env.IMAGE_NAME }}:latest # not to be pushed
```

The fewer downstream changes from the skeleton, the simpler it is to merge in new changes from the skeleton.

---

@dav3r i tried making this change before and i get this pre-commit error:
```bash
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 2

.github/workflows/build.yml
  281:46    warning  too few spaces before comment  (comments)
  ```

---

I resolved it by adding 2 spaces between the comment and the command instead of 1: [3b80f19](https://github.com/cisagov/domain-manager-ui/pull/247/commits/3b80f194568e20eed6de5b5d608b0095b93aa792)

---

Ah, sorry, that was my bad on the suggested commit - thanks for fixing that.